clients/itwin/src/Client.ts

```diff
@@ -113,6 +113,9 @@ export abstract class Client {
       } else {
         this._url = this.baseUrl;
       }
+
+      // Strip trailing '/'
+      this._url = this._url.replace(/\/$/, "");

```

Comment:
Was this happening due to the `baseUrl` already having the `/` or just making it easier if someone set the env var with a trailing slash?

Reply:
Even though the ims authority env var was being set without a slash, when constructing the URL object and adding the prefix the value I was getting back included a trailing slash and this was just the simplest solution.